File: JavaScript-Fundamentals/Homeworks/08. Functions/04. Appearance count/AppearanceCount.js
function solve(args) {
    var input = args[0].split('\n'),
        seekedNum = +input[2],
        numbers = input[1].split(' ').map(Number),
        counter = 0,
        i,
        length;

    length = numbers.length;
    for (i = 0; i < length; i += 1) {
        if (numbers[i] === seekedNum) {
            counter += 1;
        }
    }

    return counter;
}

File: Modul-I/02.C#PartTwo/Homework/Methods/AppearanceCount/AppearanceCount.cs
namespace AppearanceCount
{
    using System;
    using System.Linq;

    public class AppearanceCount
    {
        public static void Main(string[] args)
        {
            int arraySize = int.Parse(Console.ReadLine());

            int[] numbers = Console.ReadLine()
                .Split(' ')
                .Select(int.Parse)
                .ToArray();

            int seekedNumber = int.Parse(Console.ReadLine());

            int occuRanceOfSeekedNumber = SeekedNumberCount(numbers, seekedNumber);

            Console.WriteLine(occuRanceOfSeekedNumber);
        }

        private static int SeekedNumberCount(int[] numbers, int seekedNumber)
        {
            int occurances = 0;
            foreach (var number in numbers)
            {
                if (number == seekedNumber)
                {
                    occurances++;
                }
            }

            return occurances;
        }
    }
}
